README.md

```diff
@@ -0,0 +1,321 @@
+      <snapshots>
+        <enabled>true</enabled>
+        <updatePolicy>always</updatePolicy>
+      </snapshots>
+    </repository>
+  </repositories>
+</project>
+```
+
+The repository section is necessary because this project has not been published to Maven Central yet.
+
+### Getting Started
+
+Captain requires only 3 inputs from the developer (4 if you are running it in a distributed fashion). We're going to focus on the barest bones approach in this section of to see how we can run Captain on a single node. In later sections we cover all of the other features of Captain and discuss more complicated use cases.
+
+1. manifest: a list of the steps a request should follow in your workflow 
+2. config producer: a way for captain to find requests to process 
+3. request updater: a way for captain to interact with your persistence layer
+
+All it takes to get Captain running is the following:
+e.g.
+```java
+CaptainBuilder.of("example-captain-daemon", configProducer, manifestManager, requestUpdater)
+        .setNotifier(daemonNotifier)
+        .setConfigWaitTime(1, TimeUnit.SECONDS)
+        .setNextConfigWaitTime(1, TimeUnit.SECONDS)
+        .build()
+        .run();
+```
+You can find the rest of this example in ExampleCaptainWorkflow](https://github.com/LiveRamp/captain/tree/master/src/main/java/com/liveramp/captain/exception/ExampleCaptainWorkflow.java).
+
+Let's walk through what each of the components we're passing into this builder are.
+
+#### Config Producer
+
+Create a class the implements `CaptainConfigProducer`. This is your opportunity to tell Captain how to find requests to process. Commonly this will just be a db query to wherever you store requests or reading from some sort of event queue. Regardless of the implementation you're trying to pull the requests that are (or may be) ready to progress in your pipeline.
+
+e.g. If I store my requests in a db table, I'll be looking for requests that have a status of `ready`, `pending`, `in_progress` or `completed` that are not in a step of `done` or `cancelled`. Usually you're going to ignore requests that are `quarantined` or `failed` or requests that you've already completed processing (e.g. step: `done` or `cancelled`).
+
+Check out [ExampleConfigProducer](https://github.com/LiveRamp/captain/tree/master/src/main/java/com/liveramp/captain/exception/ExampleCaptainWorkflow.java) for a simple example.
+
+#### Request Updater
+
+Create a class that implements `RequestUpdater` iface. The Request Updater is your opportunity to tell Captain how it can change the step and status on your request. 
+
+In the case where you're interacting with a db or crud service, you're implementing pretty state changes like `setStatus`, `setStepAndStatus`, `quarantine`, etc.
+
+Check out [ExampleRequestUpdater](https://github.com/LiveRamp/captain/tree/master/src/main/java/com/liveramp/captain/exception/ExampleCaptainWorkflow.java) for an example.
+
+#### Manifest
```

Comment:
I feel like maybe this is better to be first? Seems like its nice to know that we have this manifest to describe the pipeline?

Reply:
so it was in the original readme. i moved it because it's really long. of the 3 top level components it's the most complex and required over 100 lines of explanation once you include its sub-components. does that change your opinion at all? if not, i'll do what you mentioned.